libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/forms/MedicalAndRehabilitationPaymentsForm/selfAssessmentSection/selfAssessmentQuestionsOneSubSection.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  buildAsyncSelectField,
+  buildDescriptionField,
+  buildMultiField,
+  buildRadioField,
+  buildSubSection,
+} from '@island.is/application/core'
+import { getYesNoOptions } from '@island.is/application/templates/social-insurance-administration-core/lib/socialInsuranceAdministrationUtils'
+import { medicalAndRehabilitationPaymentsFormMessage } from '../../../lib/messages'
+
+export const selfAssessmentQuestionsOneSubSection = buildSubSection({
+  id: 'selfAssessmentQuestionsOneSubSection',
```

Comment:
Add `tabTitle` ("New" function added in this PR: https://github.com/island-is/island.is/pull/17263)

Before:
![image](https://github.com/user-attachments/assets/86e93f6c-286e-4e01-a1ee-a7144bbc7e2f)

After:
![image](https://github.com/user-attachments/assets/a99f847e-b102-4a1a-9e2b-1d0216524390)

```suggestion
  id: 'selfAssessmentQuestionsOneSubSection',
  tabTitle:
    medicalAndRehabilitationPaymentsFormMessage.selfAssessment.sectionTitle,
```

Reply:
Added